### ISSUE TITLE
data grid

### ISSUE THREAD

**Issue by felix-bohlin** [felix-bohlin/ui#59]

**Comment by Deufel** [comment]
What are you thinking for this component? Something like this? I need something like this for a project I may be able to work on it. Have any requirements? 

<img width="778" alt="Image" src="https://github.com/user-attachments/assets/9992aadc-9e0f-473a-b959-6c1839e793e1" />

**Comment by felix-bohlin** [comment]
Yeah, very much in that vein. I hand-rolled data grids a lot before so I have respect for the effort that goes into a proper solution. But yeah, MUI-esque like that would be a great first step, for sure.

**Comment by felix-bohlin** [comment]
Awesome! Hmmm I'm not sure about start, text, end though. Basically anything can and will happen in data grid cells - your image is a good example of that. I'm not sure if it's good to do anything more than just having classes for justify and/or text-align start and end (with center being the default). Or did you have something in mind?

**Comment by Deufel** [comment]
The more I think about it, you're right. That's probably a better strategy. I think the academic/publishing world always aligns text left (start) and numbers right (end). Maybe default start makes sense?

**Comment by felix-bohlin** [comment]
Oh yeah that's true. And since start/left already is the default in CSS we might not even need classes for that.
